spec/amp-actions-and-events.md

```diff
@@ -102,10 +102,36 @@ Including: `input[type=radio]`, `input[type=checkbox]` and `select`.
   <tr>
```

Comment:
There is a line above that says: "Including: `input[type=radio]`, `input[type=checkbox]` and `select`." Let's add `range` to it explicitly as well.